BUILD.md

```diff
@@ -0,0 +1,170 @@
+# Build FreeOrion
+
+
+## Hardware and OS Requirements
+
+FreeOrion will compile on most recent Windows, Mac OS X and Linux operating
+systems. Other operating systems have reported to work, but support is not
+actively maintained by the FreeOrion developers. FreeOrion is developed for x86
+compatible processor architectures, other architectures haven't been tested.
+
+The FreeOrion source code makes heavy use of templates and requires much memory
+to compile ; 4GB RAM or more is recommended. To build FreeOrion, expect up to
+45 minutes on a Intel Core i5 system.
+
+
+## Required Software Dependencies
+
+FreeOrion depends on the following software to build:
+
+  * [Visual Studio] - 2015 for Windows Desktop Update 3 ; Windows only
+  * [Xcode] - 6.4 or later ; Mac OS X only
+  * [CMake] - 3.4 (Mac OS X) ; 3.1 or later (Other non-Windows)
+  * A C++11 compliant compiler - Other Operating Systems
+    * [GNU GCC] - 5.0 or later
+    * [Clang] - 3.9 or later
+  * [Python] - 2.7 or later, but not 3 or later
+  * [Git]
+
+FreeOrion depends on the following libraries or APIs to run:
+
+  * OpenGL - 2.1 or later ; usually provided by the graphic card driver or
+    Operating System
+  * OpenGL Utilities (GLU) ; usually provided by the graphic card driver or
+    Operating System
+  * OpenAL - It's recommended to use the [OpenAL Soft] implementation
+  * [Boost] - 1.56 or later
+  * [zlib]
+  * [libpython] - 2.7 or later, but not 3 or later
+  * [FreeType2]
+  * [libpng]
+  * [libogg]
+  * [libvorbis]
+  * [SDL2]
+
+
+## Obtaining FreeOrion Source Code and Software Dependencies
+
+For Windows and Mac OS X a [FreeOrion Software Development Kit] is provided as
+download to compile FreeOrion from source. It contains the preconfigured and
+-compiled build and runtime dependencies for Visual Studio v140 toolchain on
+Windows and Mac OS X 10.9 SDK with Xcode or later on Mac OS X.
+
+For Linux or other Operating Systems the build and runtime dependencies should
+be installed by the preferred native way (e.g. via Package manager or compiling
```

Comment:
I'm not a native speaker, but "preferred native way" sounds a bit odd. Maybe "preferred way of your distro", or "preferred way of the distro in question"?

Reply:
Although 'preferred native way' does sound reasonable to me, the other variations do sound slightly better, I'll suggest "preferred way for the respective distro"

Reply:
Also, while I can understand that you probably want to avoid having to mention too many different Linux flavors here with different details for each, it does seem a shame to not mention the sample one-liner command line for installing the dependencies via apt.

Reply:
I adopted @Dilvish-fo suggestion, replacing `distro` with `OS`.

Reply:
@Dilvish-fo 
Regarding the `apt one-liner` I avoided this on purpose, once for the reason you mentioned but also because we don't know on which distro version this `apt one-liner` is executed on.  Packages may be renamed between distro versions or the default packages are too stale for our requirements on LTS versions (eg. we need boost 1.58 on Ubuntu 16.04 LTS, which has 1.54(?) as default) so the one line is not really portable even between versions.

What would you suggest for _important_ distributions and which versions should be included into the build instructions?